### DISCUSSION TITLE
Hyper becomes very slow after updating to 3.1.0-canary

### DISCUSSION THREAD

**Discussion by Alalalalaki** [vercel/hyper#5689] [General]
It works all fine under 3.0.2 stable.

But input, output, everything response slowly after updating to 3.1.0-canary.

OS version: macOS 11.4

**Comment by yeuk0** [comment]
Try to uninstall / disable some of the plugins you have. Use `hyper list` to know exactly which ones you have.

Remove all and try to enable them one by one to see which one was causing issues (hyperline was lagging Hyper in my case).

Remember to make a backup of your settings file.

**Reply by Alalalalaki** [reply]
Hi, thanks for your advise. I find that the cause for the slow response is the hyper-statusline package.

However, I also find that even with no plugin installed, the 3.1.x-canary version has obviously longer starting time (the time to run .bash_profile?) than 3.0.2 .
